packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -50,6 +50,12 @@ Parameters:
     Default: 'false'
     AllowedValues: [ 'true', 'false' ]
     Type: String
+  BootstrapVariant:
+    Type: String
+    Default: 'AWS CDK: Default Resources'
+    Description: Describe the provenance of the resources in this bootstrap
+      stack. Change this when you customize the template. To prevent accidents,
+      the CDK CLI will not overwrite bootstrap stacks with a different variant.
```

Comment:
[nit] Maybe some copy 🍝 would come in handy here: it does not matter if it's a custom variant, other customer variant or default one, none of them can overwrite each other -- right?

Reply:
Unless `--force`